src/prototype/WingetCreateMakeDSC.ps1

```diff
@@ -0,0 +1,63 @@
+# This script is a prototype for quickly creating DSC files.
+
+if ($null -eq (Get-InstalledModule -Name Microsoft.Winget.Client))
+{
+  Install-Module Microsoft.Winget.Client
+}
+
+if ($null -eq (Get-InstalledModule -Name powershell-yaml))
+{
+  Install-Module powershell-yaml
+}
+
+[System.Collections.ArrayList]$finalPackages = @()
+$configurationVersion = "0.2.0" 
+
+$continue = $true
+while ($continue)
+{
+  $appId = Read-Host "What is the id of your app?"
+  $findResult = Find-WinGetPackage $appId
+  
+  if ($findResult.count -ne 0)
+  {
+    $index=0
+    foreach ($package in $findResult)
+    {
+        $packageDetails = "[$($index)] $($package.Name) | $($package.Id) | $($package.Version)"
+        Write-Host $packageDetails
+        $index++
+    }
```

Comment:
This can actually be achieved using the pipeline, and might be cleaner
```powershell
$findResult | ForEach-Object {$i=1} {Add-Member -InputObject $_ -NotePropertyName Index -NotePropertyValue $i; $i++ }
$findResult | Select-Object -Property Index,Name,Id,Version
```

Reply:
I wanted to get an update out, so I skipped this one for now.